src/main/scala/com/databricks/spark/xml/XmlInputFormat.scala

```diff
@@ -25,14 +25,17 @@ import org.apache.hadoop.io.{DataOutputBuffer, LongWritable, Text}
 import org.apache.hadoop.mapreduce.{InputSplit, RecordReader, TaskAttemptContext}
 import org.apache.hadoop.mapreduce.lib.input.{FileSplit, TextInputFormat}
 
+import scala.collection.mutable.ArrayBuffer
+
 /**
- * Reads records that are delimited by a specific start/end tag.
- */
+  * Reads records that are delimited by a specific start/end tag.
```

Comment:
I'd revert these whitespace changes; we've used javadoc-style indentation like Spark does. No idea why Scala did it differently.